### DISCUSSION TITLE
11.6.4

### DISCUSSION THREAD

**Discussion by krzychu124** [CitiesSkylinesMods/TMPE#1311] [Announcements]
## What's Changed
* Migrate to Harmony 2.x by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/866
* Fix "Priority roads.Tooltip:Allow far turns" string message by @TianQiBuTian in https://github.com/CitiesSkylinesMods/TMPE/pull/904
* GenericObservable - Use locks; Reuse existing list by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/909
* Skytools benchmark by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/907
* Fix SA1413 warnings by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/917
* Fix SA1003 warnings by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/918
* Remove CodeProfiler, SkyTools.Benchmark will serve the future. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/914
* Remove Finalizers by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/915
* Fix stay-in-lane for MOM tracks by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/919
* Add Unofficial.Microsoft.Unity.Analyzers nuget package by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/922
* LoopUtil.SpiralLoop replacement for better perf by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/906
* Fix SA1117 warnings by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/926
* Fix SA1201 warnings by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/927
* Fix SA1111 warnings by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/928
* Fix SA1508 warnings by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/930
* more precise lane connector overlays. by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/932
* Fix IDE0003 warnings by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/940
* Make it easier to debug harmony problems by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/933
* Apply Shift+Click speed limit to entire roundabout by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/920
* Fix warnings undo ide0003 by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/948
* Remove `#if PARKINGAI` by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/929
* Switched from Unofficial.Microsoft.Unity.Analyzers to Microsoft.Unity… by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/950
* Fix warnings mixed by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/941
* catch null segment info by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/957
* TMPE tool should behave when returning from free camera view by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/938
* Move IT support by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/956
* fixed typo in lane record by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/965
* Removed non referenced stuff marked as "// Not used" from the codebase. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/937
* post build script to create mdb file for file and line numbers by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/973
* Update BUILDING_INSTRUCTIONS.md by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/976
* fixed typo by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/981
* Updated icons for RoadSelectionPanel to optimized 8bit  by @chameleon-tbn in https://github.com/CitiesSkylinesMods/TMPE/pull/984
* Update Log.cs by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/997
* TMPE support in editor by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/990
* place intersection event by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1011
* record vehicle restrictions by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/966
* Spiral bugfix and improvement by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/994
* no double crossing policy by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1003
* use the new MoveIT integration interface by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1005
* Its not like everything must have a dot. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1022
* Deprecated SpiralLoop cleanup. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1019
* Removed ProcessLane method, uint.ToLane() is used instead. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1028
* Removed ProcessBuilding method that used a closure. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1027
* Replaced ProcessNode(..) with ToNode() to remove closures. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1029
* Removed NetService.ProcessSegment by inlining the NetManager call. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1023
* Bugfix floating vehicles when Parking AI disabled by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/993
* Right click to close main menu or exit current tool by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1032
* Monitor.Enter/Exit changes by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1048
* SpeedLimits stage 1: U interface and texture loading changed by @kvakvs in https://github.com/CitiesSkylinesMods/TMPE/pull/1044
* its not like anyone will miss an empty project right? by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1056
* old stuff that should have been deleted earlier. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1057
* Fixed all instances of "SA1507 Code should not contain multiple blank… by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1058
* Unity profiler support with various improvements by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1054
* Closure killer for IterateNodeSegments by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1055
* New languages, update existing by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1047
* CitiesHarmony API upgrade by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1059
* cleaning up life cycle code by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1068
* Fixed SA1410 warnings by replacing the delegates with lambdas. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1069
* Fixed SA1137 Elements should have the same indentation warnings. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1070
* Fixed all instances of delegate instance allocation (HAA0603) where i… by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1071
* This is just a 11.5.1-hotfix1 merge with strategy=ours into master. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1073
* Migration to Harmony by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1060
* Hide overlays if no longer necessary by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1080
* Main Camera cache reinitialization to prevent use of inactive camera by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1079
* 730 cleanup loading custom path manager by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1072
* harmon patcher continues on error by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1076
* lifecycle class by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1077
* Converted .ruleset to .editorconfig by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1082
* Chinese icon by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1081
* Removed some documentation warnings. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1083
* Fixed issue in LaneConnectorTool ... by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1086
* Master to testing by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1088
* leftovers from .ruleset to .editorconfig conversion. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1087
* Fixed all cases of Warning SA1514 Element documentation header should… by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1085
* Migration to Harmony by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1089
* Translation update by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1090
* Moved foreach even one further level out to fix the bug with multiple… by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1094
* More Vehicles compatibility fix by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1093
* Citizen not found fix by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1091
* Stable 11.5.2 cargo truck PF fix by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1105
* 1095 asset lifecycle by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1096
* dedicated turning lane policy by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1104
* 11.5.2 to master by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1108
* Rename LABS to TEST, prepare release to the Workshop by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1098
* 11.5.2-hotfix-2 to master by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1113
* undid path updates when deserializing. by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1117
* Update locale specific textures on locale change by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1121
* flag extension by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1118
* Lane connection overlay and performance improvements by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/985
* CustomPathFind reflection calls by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1114
* Ignore invalid (disconnected) segments while calculating lane routings by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1125
* fix Infinite loop rabout by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1131
* Notifier by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1127
* Opacity for legacy subTool GUI by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1129
* Translation update, Indonesian translation added by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1135
* Improved half-segment detection by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1141
* Batch 5 of Speedlimits UI: Changes to U library by @kvakvs in https://github.com/CitiesSkylinesMods/TMPE/pull/1143
* Incomplete vanilla DLC detection fix by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1144
* refresh dedicated turning lanes on load by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1140
* Incorrect height of vehicle restriction signs by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1142
* Improved pathfinding lifecycle by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1145
* EnumerateSpeedLimits in OnToolGUI .... by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1149
* Simplification - Moved BuildingService methods into ExtBuildingManager. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1150
* Minor benchmark stuff by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1155
* Removed weird foreach over { false, true } hack. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1154
* configurable log file path. by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1151
* Removed weird nested ForEachLane method. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1156
* Scale unity GUI by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1152
* Simplify UI scaler by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1157
* UUI integration by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1153
* Replace slow, garbage generating `TinyDictionary` with regular one by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1159
* No need to create a new list just so you can use List<T>.IndexOf(..) … by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1161
* Simpler method, since we don't need a lambda here. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1164
* Closure killer iterate segment lanes by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1162
* Bugfix - Fixed issue in ResetIncomingHighwayLaneArrows introduced due… by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1165
* Cars and cargo trucks despawn despite the Despawning is off by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1167
* Batch 6: Speed Limits manager now uses structures with intent by @kvakvs in https://github.com/CitiesSkylinesMods/TMPE/pull/1163
* Cleaned up ResetIncomingHighwayLaneArrows a little bit. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1166
* Removed the ProcessCitizen and the CitizenHandler to remove unnecessa… by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1169
* Removed ProcessCitizenInstance method and the CitizenInstanceHandler … by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1170
* Removed ReleaseCitizenInstance indirection for simplicity. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1171
* Removed IsCitizenValid indirection for simplicity. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1172
* Refactoring citizenservice by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1174
* Removed TinyDictionary completely after krzy already removed it from … by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1177
* Fixed previous rename in debug configuration. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1178
* Inlined NetService.IsNodeValid into ExtNodeManager.IsValid and fixed … by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1175
* Killed of SimulationService indirection. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1179
* Removed VehicleService - see individual commits. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1176
* update UUI version by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1182
* Refactoring/removenetservice by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1183
* Switched from CO.IsFlagSet to Kian's IsFlagSet. by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1186
* Trains not counted as waiting - Timed Traffic Lights by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1184
* Refactoring/removenetservice by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1185
* guarded against null when disabling tool by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1187
* Priority signs tool not working by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1189
* Replace hardcoded CitizenUnit buffer limits with buffer size detection by @algernon-A in https://github.com/CitiesSkylinesMods/TMPE/pull/1192
* improve lifecycle of TMPE tool by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1188
* Incompatible detector improvements by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1196
* Remove NetService and the GameBridge stuff by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1194
* Fixed recalculation after lane connection remove by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1199
* Refactoring/IsValid by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1197
* Fix typo in `IdToSegmentWithIsValidExtension()` by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1201
* Refactoring/buffer to extension methods by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1203
* GetNode/Segment -> ToNode/Segment by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1202
* log exception stack trace for Traffic rules record. by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1200
* Converted almost all .m_segments.m_buffer[segmentId] calls to segment… by @DaEgi01 in https://github.com/CitiesSkylinesMods/TMPE/pull/1204
* fix: routing manager lane connection check by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1210
* More API by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1209
* fix uturn policy on T junctions by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1212
* fixed JRM transparency by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1217
* New Speed Limits UI by @kvakvs in https://github.com/CitiesSkylinesMods/TMPE/pull/1168
* Update `SPEED_TO_MPH` constant to correct value by @aubergine10 in https://github.com/CitiesSkylinesMods/TMPE/pull/1224
* Adds checkbox label text wrap for some very long options by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1232
* fix position of lane arrow tool for different resolutions. by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1237
* update routings after modifying dedicated turning lanes policy by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1236
* fixed regression: Auto TTL should set-up dedicated turning lanes properly by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1242
* Apply default speed limits to all elevations by @kvakvs in https://github.com/CitiesSkylinesMods/TMPE/pull/1233
* Road sign themes reloadable + Canadian signs by @kvakvs in https://github.com/CitiesSkylinesMods/TMPE/pull/1215
* New countries for speed limit designs by @kvakvs in https://github.com/CitiesSkylinesMods/TMPE/pull/1260
* Static texture classes are now proper CustomManagers by @kvakvs in https://github.com/CitiesSkylinesMods/TMPE/pull/1265
* Netutil cleanup by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1273
* Speed Limits-cleanup by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1275
* Version bump - 11.6.2 by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1278
* versioned atlas by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1281
* Add 2 Traffic++ v2 mods to incompatible list by @aubergine10 in https://github.com/CitiesSkylinesMods/TMPE/pull/1287
* Better AssetEditor mode detection by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1286
* Slm data rework by @kianzarrin in https://github.com/CitiesSkylinesMods/TMPE/pull/1277
* Play mode detection improvement by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1290
* What's New panel by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1284
* Readme 11.6.3 by @aubergine10 in https://github.com/CitiesSkylinesMods/TMPE/pull/1293
* Changelog v11.6.3 by @aubergine10 in https://github.com/CitiesSkylinesMods/TMPE/pull/1292
* Reduce loading log spam from AssetDataExtension by @aubergine10 in https://github.com/CitiesSkylinesMods/TMPE/pull/1296
* TM:PE settings not copy pasted in-game by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1303
* Update localization files by @aubergine10 in https://github.com/CitiesSkylinesMods/TMPE/pull/1304
* General tab - redux by @aubergine10 in https://github.com/CitiesSkylinesMods/TMPE/pull/1306
* Place Intersection improvements, bugfixing  by @krzychu124 in https://github.com/CitiesSkylinesMods/TMPE/pull/1308
* Fix game version warning by @aubergine10 in https://github.com/CitiesSkylinesMods/TMPE/pull/1310
* 11.6.4 What's New by @aubergine10 in https://github.com/CitiesSkylinesMods/TMPE/pull/1305

## New Contributors
* @DaEgi01 made their first contribution in https://github.com/CitiesSkylinesMods/TMPE/pull/909
* @chameleon-tbn made their first contribution in https://github.com/CitiesSkylinesMods/TMPE/pull/984
* @algernon-A made their first contribution in https://github.com/CitiesSkylinesMods/TMPE/pull/1192

**Full Changelog**: https://github.com/CitiesSkylinesMods/TMPE/compare/11.5.2...11.6.4

<hr /><em>This discussion was created from the release <a href='https://github.com/CitiesSkylinesMods/TMPE/releases/tag/11.6.4'>11.6.4</a>.</em>

**Comment by originalfoo** [comment]
@krzychu124 pls update mod checker link on TEST/STABLE workshop pages to point to Compatibility Report mod: https://steamcommunity.com/sharedfiles/filedetails/?id=2633433869

EDIT: On the STABLE page, it might be worth moving the "this version 100% compatible with TM:PE 10.20" box nearer end of page?

**Reply by krzychu124** [reply]
It took me a while to recall about this small task😄 

> On the STABLE page, it might be worth moving the "this version 100% compatible with TM:PE 10.20" box nearer end of page?

Like where? Between "changelog" and link to Code&translations?

**Reply by originalfoo** [reply]
Yeah, something like this:

![img](https://imgur.com/yCNCctg.png)
![img](https://imgur.com/KDMGpYR.png)
![img](https://imgur.com/TXqAFNM.png)
![img](https://imgur.com/8h5I3Pe.png)
![img](https://imgur.com/LtuHhXs.png)
![img](https://imgur.com/LHw7mT1.png)
![img](https://imgur.com/NSKgXo3.png)
![img](https://imgur.com/5mUvoKa.png)
![img](https://imgur.com/Rs9ETjA.png)
![img](https://imgur.com/KTffR5h.png)

In migration panel, I'd ditch the blue heading entirely tbh. Change the opening text to smth like:

> **This is STABLE release with reliable features.**
>  
> When switching to/from other TM:PE versions:
> ....

**Reply by krzychu124** [reply]
done - more or less 😅 OK... less
